internal/executor/rpc/rpc.go

```diff
@@ -227,6 +226,24 @@ func (r *RPC) ReportSingleCommand(
 	return &api.ReportSingleCommandResponse{}, nil
 }
 
+func (r *RPC) getCommandLogger(task *build.Task, commandName string) *echelon.Logger {
+	commandLoggerScope := fmt.Sprintf("'%s'", commandName)
+	// make pretty names for some instruction types
+	command := task.GetCommand(commandName)
+	switch {
+	case command == nil:
+		// no found :-(
+		break
+	case command.ProtoCommand.GetScriptInstruction() != nil:
+		commandLoggerScope += " script"
+	case command.ProtoCommand.GetBackgroundScriptInstruction() != nil:
+		commandLoggerScope += " background script"
+	case command.ProtoCommand.GetCacheInstruction() != nil:
+		commandLoggerScope += " cache"
+	}
```

Comment:
I've ran this branch against CLI's `.cirrus.yml` and it seems that we can add the artifacts instruction here too.

Reply:
see e250602468305ed08de4ed48ec72139611945162